draft-ietf-httpbis-variants.md

```diff
@@ -525,7 +597,7 @@ To perform content negotiation for Accept-Encoding given a request-value and ava
 
 1. Let preferred-available be an empty list.
 2. Let preferred-codings be a list of the codings in the request-value, ordered by their weight, highest to lowest, as per Section 5.3.1 of {{!RFC7231}} (omitting any coding with a weight of 0). If "Accept-Encoding" is not present or empty, preferred-codings will be empty. If a coding lacks an explicit weight, an implementation MAY assign one.
-3. If "identity" is not a member of preferred-codings, append "identity".
+3. If "identity" is not a member of preferred-codings, and it is not specifically excluded by the request-value stating either "identity;q=0" or "*;q=0" without a more specific entry for "identity", append "identity" to preferred-codings.
```

Comment:
Encoding is not different; the server can ignore the client's preferences here too.

Reply:
That is, I should undo this change? That's fine with me.

Reply:
Yes.